src/towncrier/test/test_build.py

```diff
@@ -511,6 +511,18 @@ def test_needs_config(self):
         self.assertEqual(1, result.exit_code, result.output)
         self.assertTrue(result.output.startswith("No configuration file found."))
 
+    @with_isolated_runner
+    def test_needs_version(self, runner: CliRunner):
+        """
+        If the configuration file doesn't specify a version or a package, the version
+        option is required.
+        """
+        with open("towncrier.toml", "w") as f:
+            f.write("[tool.towncrier]")
+        result = runner.invoke(_main, ["--draft"], catch_exceptions=False)
```

Comment:
Let's leave some space between arrange, act, and assert.

```suggestion

        result = runner.invoke(_main, ["--draft"], catch_exceptions=False)

```